frontend/src/test/scala/bloop/BaseCompileSpec.scala

```diff
@@ -1543,6 +1543,78 @@ abstract class BaseCompileSpec extends bloop.testing.BaseSuite {
     }
   }
 
+  test("Recompile entire mixed project") {
```

Comment:
Great test! Can we use remove the capital case from the test though? We use lowercase in the whole codebase.